v1/src/main/java/com/google/cloud/teleport/spanner/common/SizedType.java

```diff
@@ -49,6 +49,10 @@ public SizedType(Type type, Integer size, Integer arrayLength) {
   }
 
   public static String typeString(Type type, Integer size) {
+    return typeString(type, size, /*outputAsDdlRepresentation=*/false);
+  }
+
+  public static String typeString(Type type, Integer size, boolean outputAsDdlRepresentation) {
```

Comment:
Can you add a detailed comment on what is outputAsDdlRepresentation (true, and false) with examples? It'll help for other developers understand and avoid confusion.

Reply:
Done